src/monster.cpp

```diff
@@ -621,6 +623,8 @@ int monster::print_info( const catacurses::window &w, int vStart, int vLines, in
         mvwprintz( w, point( column, vStart++ ), c_white, _( "Rider: %s" ), mounted_player->disp_name() );
     }
 
+    wprintz( w, c_light_gray, _( " It is " ) + size_names.at( get_size() ) + _( "." ) );
```

Comment:
```suggestion
    wprintz( w, c_light_gray, _( " It is %s." ), size_names.at( get_size() ) );
```
Translators will have massive problems translating a string like "." as it has basically no context.